### PR TITLE
Fix the commit regex to account for carriage returns

### PR DESCRIPTION
**Pull request by DilumAluthge**

### CODE REVIEW

**Comment by DilumAluthge** [comment]
cc: @ericphanson @GunnarFarneback

**Comment by giordano** [comment]
Out of curiosity, where is the carriage return in https://github.com/JuliaRegistries/General/pull/36450?

File: src/AutoMerge/pull-requests.jl [hunk 1]
```diff
@@ -2,7 +2,7 @@ const new_package_title_regex = r"^New package: (\w*?) v(\S*?)$"
 
 const new_version_title_regex = r"^New version: (\w*?) v(\S*?)$"
 
-const commit_regex = r"(?:^|\n)(?:\-|\*) Commit: (\w*?)(?:$|\n)"
+const commit_regex = r"(?:^|\n|\r\n)(?:\-|\*) Commit: (\w*?)(?:$|\n|\r\n)"
```
[the remark this answers is not in the document]
Comment (DilumAluthge) [reply src/AutoMerge/pull-requests.jl:5]: Ah, Bors just merged. Want to make a followup PR to make this simplification?

**Comment by giordano** [comment]
> For some reason, the GitHub web editor must have inserted them.

Great :expressionless:

File: src/AutoMerge/pull-requests.jl (hunk 1 above)
[the remark this answers is not in the document]
Comment (giordano) [reply src/AutoMerge/pull-requests.jl:5]: No, I don't think it's worth a new pull request :slightly_smiling_face:

**Comment by ericphanson** [comment]
Thanks for the fix @DilumAluthge!

I think our “staging” workflow on General would’ve caught this too so we can use that next time also. I think probably I should’ve turned it back on once the BinaryBuilder PR was merged.

**Comment by DilumAluthge** [comment]
Actually, it might not have caught it..... I think the reason the carriage returns were there was that @giordano edited the PR body using the GitHub web editor, and GitHub added those carriage returns without telling anyone.

**Comment by DilumAluthge** [comment]
Or maybe his web browser added the carriage returns?

**Comment by ericphanson** [comment]
Ahh, I missed that, very sneaky of GitHub.... I saw somewhere they also do some weird unicode normalization when you edit a file with their web editor (globally in the file, not just the bit you edit)...

**Comment by giordano** [comment]
> I saw somewhere they also do some weird unicode normalization when you edit a file with their web editor (globally in the file, not just the bit you edit)...

https://github.com/JuliaDiff/ChainRules.jl/pull/403#issuecomment-828441781

**Pull request merged**